dist/combobox/ds6/combobox.css

```diff
@@ -95,6 +95,18 @@ span.combobox {
 .combobox__option[role^="option"]:active {
   font-weight: bold;
 }
+.combobox__option[role^="option"]:first-child {
```

Comment:
what is the `[role^="option"]`  syntax called again? I need to learn more about this